recipes/gwham/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "gwham" %}
+{% set version = "2.0.10.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://membrane.urmc.rochester.edu/sites/default/files/wham/wham-release-{{ version }}.tgz
+  sha256: 44635a429ded5892950271fa58e97691628ad7a3be21d09f86ca2830e2b679f7
+  folder: wham_{{ version }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+
+test:
+  commands:
+    - "./wham/wham -h"
+    - "./wham-2d/wham-2d -h"
+
+about:
+  home: http://membrane.urmc.rochester.edu/?page_id=126
+  license: GPL-3.0-or-later
+  license_family: GPL
```

Comment:
You're going to want to manually package the license file if it isn't in the tarball:

https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually